src/main/features/core/applicationMenu.js

```diff
@@ -104,7 +104,9 @@ const template = [
 
 if (process.platform === 'darwin') {
   template.unshift({
-    label: (global.DEV_MODE)? 'Electron' : 'Google Play Music Desktop Player',
+    label: (global.DEV_MODE && process.platform === 'darwin') ?
```

Comment:
Couple of things here

1. The check for `darwin` is pointless because literally 2 lines above we are checking for `darwin`
2. `global.DEV_MODE` being `true` does **NOT** guarantee the application name is Electron or GPMDP.  We should use `app.getName()`
3. But most importantly this is overflowing the scope of this PR.  See the [CONTRIB.md](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/CONTRIBUTING.md#pull-requests) :cry: